Stream/UsageCost/usage-cost-stream.yaml

```diff
@@ -0,0 +1,61 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: usage-sender
+  labels:
+    app: usage-cost-stream
+spec:
+  containers:
+    - name: usage-sender
+      image: hananiel/usagesender:latest
```

Comment:
Are the images in here available for anybody to pull? Should this be a variable/script combo instead so they can be build/pushed locally?

Reply:
I'll update it to the vmware harbor repo thats publicly available. In the docs themselves I replaced this with <your-repo> since I had the instructions building from dockerfile and pushing to <your-repo>